core/src/apps/cardano/layout.py

```diff
@@ -130,6 +145,48 @@ async def confirm_transaction(ctx, amount: int, fee: int, protocol_magic: int):
     await require_hold_to_confirm(ctx, Paginated([t1, t2]))
 
 
+async def confirm_certificate(
+    ctx: wire.Context, certificate: CardanoTxCertificateType
+) -> bool:
+    pages = []
+
+    t1 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+    t1.normal("Confirm:")
+    t1.bold(CERTIFICATE_TYPE_NAMES[certificate.type])
+    t1.normal("for account:")
+    t1.bold(address_n_to_str(to_account_path(certificate.path)))
+    pages.append(t1)
+
+    if certificate.type == CardanoCertificateType.STAKE_DELEGATION:
+        t2 = Text("Confirm transaction", ui.ICON_SEND, ui.GREEN)
+        t2.normal("to pool:")
+        t2.bold(hexlify(certificate.pool).decode())
```

Comment:
You can use `certificate.pool.hex()`.

Reply:
that does not exist in micropython IIRC

Reply:
I thought it was added few versions back but it seems I am wrong. Maybe it was `decode`? Anyway, ignore then..